src/Command/CleanupSnapshotsCommand.php

```diff
@@ -71,7 +71,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     }
 
     /**
-     * @param array<int> $ids
+     * @param array<int, string> $ids
```

Comment:
The key of this array doesn't matter in this method.

Did you meant array<int|string> ?

Reply:
I mean: array<string>. It is an argument from the console, so this array is made of strings.

Reply:
> I mean: `array<string>`. It is an argument from the console, so this array is made of strings.

should we convert string to int and require `array<int>`?

Reply:
I don't know if that is a good idea. Let's discuss that as a separate issue. (The user defines the primary key, and it can be defined as string if he wants, so doing what you say it could be a problem).